js/functions.ts

```diff
@@ -11,4 +11,14 @@ const getNumber = (string:string) => {
 	return parseInt(string.replace(/[^0-9]/g,''), 10);
 };
 
-export {lessOrEqual, isPalindrome, getNumber};
+// задание 11
+const parseTime = (time:string) => {
+	const parts = time.split(':').map(Number);
+	const [hours, minutes] = parts;
+	const minutesPerHour = 60;
+	return hours * minutesPerHour + minutes;
+};
```

Comment:
Вот эта переменная нам известна заранее и нет смысла создавать её на каждый вызов. Это общая информация, поэтому стоит оформит как константу и вынести на верхний уровень
```suggestion
const MINUTES_PER_HOUR = 60; // в начало модуля после импортов для лучшего код гайда

const parseTime = (time:string) => {
	const parts = time.split(':').map(Number);
	const [hours, minutes] = parts;
	return hours * MINUTES_PER_HOUR + minutes;
};
```